server/router_utils.go

```diff
@@ -83,7 +83,8 @@ func readCompositeRuleID(writer http.ResponseWriter, request *http.Request) (
 		return
 	}
 
-	ruleID = ctypes.RuleID(ruleIDParam)
+	//trim the .report from module part if present
```

Comment:
space :)

Reply:
True, true. Adding that.
BTW the `writer` argument is not used, so I'll remove it from the function.